app/scripts/controllers/network/network-controller.test.js

```diff
@@ -899,25 +898,18 @@ describe('NetworkController', () => {
                   type: 'rpc',
                   rpcUrl: 'https://mock-rpc-url',
                   chainId: '0x1337',
-                  nickname: 'test-chain',
-                  ticker: 'TEST',
-                  rpcPrefs: {
-                    blockExplorerUrl: 'test-block-explorer.com',
-                  },
-                  id: 'testNetworkConfigurationId',
-                },
-                networkConfigurations: {
```

Comment:
`setProviderType` doesn't need a network configuration to be in place in order to work, so it is unnecessary in this test. The set of changes in this test were made so that the block tracker wouldn't complain that it couldn't make a request (requests aren't being properly mocked here). The part that you highlighted was just a cleanup sort of thing because the network configuration object isn't actually necessary in this test. 

That said, these changes aren't necessary at all to make the tests pass. They will be removed in #17556 anyway.